LayoutTests/http/tests/paymentrequest/paymentrequest-applePayLaterAvailability.https.html

```diff
@@ -46,7 +46,7 @@
 
 user_activation_test(async (test) => {
     const method = validPaymentMethod();
-    method.data.applePayLaterAvailability = "unavailableMerchantIneligible";
+    method.data.applePayLaterAvailability = "UnavailableRecurringTransaction";
```

Comment:
Is this case sensitive? The enum value in the IDL below is `unavailableRecurringTransaction`.

Reply:
Oop it definitely is. Conflated the WebCore enum with the IDL enum, turns out they have different casing. Fixed in https://github.com/WebKit/WebKit/commit/a1800a4bbcfbb2ef1dac06187087415cbae58da6